__main__.py

```diff
@@ -0,0 +1,216 @@
+    if stdout.isatty():
+      print(f'\x1B[1;44m{first:78}\x1B[0m\n{rest}')
+    else:
+      print(f'{first}\n{rest}')
+    sys.exit(0)
+  if any(a.lower() in ('-v', '--version') for a in av):
+    print(cmdhelp.split('\n')[0])
+    sys.exit(0)
+  ad = {}
+  args = Args()
+  # Support a few other forms for Age etc. compatibility (but only as the first arg)
+  if av[0].lower() in ('enc', 'encrypt', '-e', '--encrypt'):
+    args.mode, ad, modehelp = 'enc', encargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
+  elif av[0].lower() in ('dec', 'decrypt', '-d', '--decrypt'):
+    args.mode, ad, modehelp = 'dec', decargs, f"{hdrhelp}\nEncryption options:\n{enchelp}"
+  elif av[0].lower() in ('bench', 'benchmark'):
+    args.mode, ad, modehelp = 'benchmark', benchargs, f"{hdrhelp}"
+  elif next((k for k, v in encargs.items() if av[0] in v), None) is not None:
+    args.mode, ad, modehelp = 'enc', encargs, enchelp
+    av.insert(0, None)
+  else:
+    stderr.write(' 💣  Invalid or missing command (enc/dec/benchmark).\n')
+    sys.exit(1)
+
+  aiter = iter(av[1:])
+  longargs = [flag[1:] for switches in ad.values() for flag in switches if flag.startswith("--")]
+  shortargs = [flag[1:] for switches in ad.values() for flag in switches if not flag.startswith("--")]
+  print(shortargs)
+  for a in aiter:
+    aprint = a
+    if not a.startswith('-'):
+      args.files.append(a)
+      continue
+    if a == '-':
+      args.files.append(True)
+      continue
+    if a == '--':
+      args.files += args
+      break
+    if a.startswith('--'):
+      a = a.lower()
+    if not a.startswith('--') and len(a) > 2:
+      if a not in longargs:
+        if any(longarg[1:] in a for longarg in longargs) is True:
+          stderr.write(f' 💣  Multiple arguments cannot have long arguments: covert {args.mode} {a}\n')
+          sys.exit(1)
+        elif any(arg not in shortargs for arg in list(a[1:])) is True:
+          falseargs = [arg for arg in list(a[1:]) if arg not in shortargs]
+          stderr.write(f' 💣  {falseargs} is not an argument: covert {args.mode} {a}\n')
+          sys.exit(1)
+        a = [f'-{shortarg}' for shortarg in list(a[1:]) if shortarg in shortargs]
+      else:
+        a = f'-{a}'
+    if isinstance(a, str) is True:
+      a = [a]
+    for i, av in enumerate(a):
+      argvar = next((k for k, v in ad.items() if av in v), None)
+      if isinstance(av, int) is True:
+        continue
+      if av is None:
+        stderr.write(f'{modehelp}\n 💣  Unknown argument: covert {args.mode} {aprint}\n')
+        sys.exit(1)
+      try:
+        var = getattr(args, argvar)
+        if isinstance(var, list):
+          var.append(next(aiter))
+        elif isinstance(var, str):
+          setattr(args, argvar, next(aiter))
+        elif isinstance(var, int):
+          setattr(args, argvar, var + 1)
+        else:
+          setattr(args, argvar, True)
+        print(var)
+        if isinstance(var, int) is not True and var is not None:
```

Comment:
```python
if not isinstance(var, int) and var is not None:
```

Reply:
Looks like what you really wanted here is `isinstance(var, list)` or `isinstance(var, (str, list))` but as commented below, this entire check should be removed.